doc-Managing_Authentication/topics/External_auth.adoc

```diff
@@ -1,25 +1,26 @@
 [[external_auth]]
-= Configuring Federated Identity Management (External Authentication) with {product-title_short}
+= Configuring Identity Management (External Authentication) with {product-title_short}
 
-//Overview of federated auth in CloudForms here - benefits
 
-You can configure {product-title_short} to use federation authentication with system authentication methods such as Red Hat Identity Management (IdM), Red Hat Single Sign-On (SSO), or Active Directory (AD). 
+You can configure {product-title_short} to use federation authentication with system authentication methods such as Red Hat Identity Management (IdM) or IPA, Red Hat Single Sign-On (SSO), or Active Directory (AD). 
+
+This method uses `apache` (`httpd`) modules with web browsers to control authentication to {product-title_short}. It is the recommended authentication method to connect {product-title_short} with most identity management services.
 
 [[external_ipa_auth]]
-== Configuring Federated Authentication with IdM 
+== Configuring Federated Authentication with IPA 
 
-You can configure {product-title_short} to use Red Hat Identity Management (IdM) for federated authentication using the *External Authentication (httpd)* option in {product-title_short}.
+You can configure {product-title_short} to use IPA for federated authentication using the *External Authentication (httpd)* option in {product-title_short}.
 
-When external authentication is enabled, users can log in to the {product-title_short} appliance using their IdM server credentials. The appliance creates user accounts automatically and imports relevant information from the IdM server.
+When external authentication is enabled, users can log in to the {product-title_short} appliance using their IPA server credentials. The appliance creates user accounts automatically and imports relevant information from the IPA server.
 
-The appliance contains IdM client software for connecting to IdM servers, but it is not configured by default. External authentication is enabled by first configuring it in the web interface, then in the console.
+The appliance contains ipa client software for connecting to IPA servers, but it is not configured by default. External authentication is enabled by first configuring it in the web interface, then in the console.
```

Comment:
- IPA client software? 
- Also, did we arrive at a decision about user interface - "web user interface"? A couple more instances below of user interface that need to be made consistent.
- ..appliance console?

Reply:
I always configure external auth with first, then configure it in the MiQ web interface second but the order doesn't really matter.

_ipa_ should be _IPA_

Maybe:
The appliance contains IPA client software for connecting to IPA servers, but it is not configured by default. External authentication is enabled by configuring it with the
appliance_console and enabling it the web interface.